### DISCUSSION TITLE
Styling State - Page Load

### DISCUSSION THREAD

**Discussion by eswanzey** [vanjarosoftware/Vanjaro.Platform#1058] [Q&A]
How will object animations on page load be handled? At the moment, I'm not seeing a way to set them via the Vanjaro UI. For example, having a text block move into position when revealed during a page scroll.

**Comment by meetmandeep** [comment]
Animations are not part of the core platform. They come and go and the underlying technologies keep evolving. 

Having said that, we've already incorporated animations in our Premium Theme along with many other cool features such as Shapes & Dividers. Premium Theme will be available after official Vanjaro Release when we introduce the Vanjaro Store.
